buffer/src/main/java/io/netty5/buffer/api/internal/JniBufferAccess.java

```diff
@@ -40,7 +40,20 @@
             isAvailable = true;
         } catch (Throwable e) {
             InternalLogger logger = InternalLoggerFactory.getInstance(JniBufferAccess.class);
-            logger.debug("JNI bypass for accessing native address of DirectByteBuffer is unavailable.", e);
+            logger.debug("JNI unix bypass for accessing native address of DirectByteBuffer is unavailable.", e);
+        }
+        if (!isAvailable) {
+            try {
+                Class<?> bufferAccess = Class.forName("io.netty.internal.tcnative.Buffer");
+                Lookup lookup = MethodHandles.lookup();
+                bufferAccess = lookup.accessClass(bufferAccess);
+                MethodType type = MethodType.methodType(long.class, ByteBuffer.class);
+                memoryAddress = lookup.findStatic(bufferAccess, "address", type);
+                isAvailable = true;
+            } catch (Throwable e) {
+                InternalLogger logger = InternalLoggerFactory.getInstance(JniBufferAccess.class);
```

Comment:
nit: You do the same `InternalLogger` lookup above. Why not store it in a static field and use the same instance everywhere without extra lookup ?